src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(param) {
+      calculator.result += param;
+    },
+
+    subtract(param) {
+      calculator.result -= param;
+    },
+
+    multiply(param) {
+      calculator.result *= param;
+    },
+
+    divide(param) {
+      calculator.result /= param;
```

Comment:
There is a potential issue in the `divide` method. Dividing by zero will result in `Infinity` or `NaN`, which might not be the desired behavior. Consider adding a check to handle division by zero appropriately.